alcs-frontend/src/app/shared/inline-datepicker/inline-datepicker.component.scss

```diff
@@ -21,7 +21,6 @@
 .button-container {
   display: flex;
   justify-content: flex-end;
-  margin-right: -14px;
```

Comment:
This was added to have the buttons line up with the right side of the date picker, why are you removing it?

Reply:
This was a request from Sandra, since with -14px the button text was aligned but button outline not. So we agreed to have the outline aligned instead of text